livekit-plugins/livekit-plugins-cartesia/livekit/plugins/cartesia/tts.py

```diff
@@ -148,6 +148,11 @@ def _ensure_session(self) -> aiohttp.ClientSession:
 
         return self._session
 
+    async def prewarm(self) -> None:
+        async with self._pool.connection():
+            # open the connection and return it back to the pool
+            pass
```

Comment:
I would say to remove it from this PR if this is a TODO

Reply:
it's not.. this prewarms the connection (by grabbing a connection and returning it to the pool)

alternatively I could add a `.prewarm()` function on the pool to make this less confusing

Reply:
ahhh, nevermind then

It's because you said

> added a tts.prewarm method to start the connection pool early. we should do this automatically in v1. currently just the stubs

Reply:
looks good, VoiceAgent.start should call it automatically. but feel free to ignore for now